### PR TITLE
Improved article. Details on Dogecoin Core, adjusted some inaccuracies

### PR DESCRIPTION
**Pull request by MishaBoar**
I have fixed some inaccuracies in the original article and expanded some parts of it. For future version, we might include also details to allow users to dump their private keys from Dogecoin Core, directly - but I think for this iteration, this is good.

### CODE REVIEW

File: content/dogepedia/articles/how-to-backup-a-wallet.md [hunk 1]
```diff
@@ -5,16 +5,26 @@ date = "2021-10-22"
...
+Dogecoin Core, the software used to run a Dogecoin node, is often used as a wallet application. In order to create a backup with Dogecoin Core, follow these steps:
 
-This is why you should never share your seed phrase or private key with anyone that you do not trust. Remember, too, that Dogecoin transactions are permanent. Therefore regularly ‘backing up’ your seed phrase in a safe, secret, and secure location is very important. **It is always more secure to ensure that your seed phrase is not stored on any device that can be connected to the internet.**  
+- (Optional, but recommended) First, encrypt you wallet. To do so, go to Settings -> Encrypt wallet. You will be asked to enter a password. It is crucial that you store this password somewhere safe and that you never lose it. Without it, you will be unable to access the funds in your encrypted wallet. On the other hand, this password adds an extra layer of security to your backup. Without it, even if an attacker gains access to your wallet file, he will still be unable to spend your Dogecoin.
```
Comment (RichieJohn) [inline content/dogepedia/articles/how-to-backup-a-wallet.md:18]: Change: "... encrypt you wallet" to "encrypt **your** wallet"

**Review by RichieJohn — COMMENTED** [review]
Great! Just one typo spotted. :)

**Comment by MishaBoar** [comment]
> Great! Just one typo spotted. :)

Thanks! Fixed

**Review by RichieJohn — APPROVED** [review]
Great! :)

**Pull request merged**